pkg/graveler/graveler.go

```diff
@@ -191,6 +192,18 @@ type Repository struct {
 	StorageNamespace StorageNamespace `db:"storage_namespace"`
 	CreationDate     time.Time        `db:"creation_date"`
 	DefaultBranchID  BranchID         `db:"default_branch"`
+	State            RepositoryState
+	InstanceUID      string
```

Comment:
We need documenetation for these non-straightforward properties.

Reply:
I think the convention we're trying to hold is always to treat time in UTC. @nopcoder correct me if I'm wrong

Reply:
Done